docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -273,7 +275,681 @@ export default async function auth(req, res) {
+```
+
+A number of options are also available, enabling you to filter for certain
+rooms.
+
+```ts
+const { data: rooms, nextPage } = await liveblocks.getRooms({
+  // Optional, the amount of rooms to load, between 1 and 100, defaults to 20
+  limit: 20,
+
+  // Optional, filter for rooms that allow entry to group ID(s) in `groupAccesses`
+  groupIds: ["engineering", "design"],
+
+  // Optional, filter for rooms that allow entry to a user's ID in `usersAccesses`
+  userId: "my-user-id",
+
+  // Optional, filter for rooms with custom metadata in `metadata`
+  metadata: {
+    myRoomType: "whiteboard",
+  },
+
+  // Optional, cursor used for pagination, use `nextPage` from the previous page's response
+  startingAfter: "L3YyL3Jvb21z...",
+});
+```
+
+You can use `nextPage` to paginate rooms. In this example, when `getNextPage` is
+called, the next set of rooms is added to `pages`.
+
+```ts
+import { RoomInfo } from "@liveblocks/node";
+
+// An array of pages, each containing a list of retrieved rooms
+const pages: RoomInfo[][] = [];
+
+// Holds the pagination cursor for the next set of rooms
+let startingAfter;
+
+// Call to get the next page of rooms
+async function getNextPage() {
+  const { data, nextPage } = await liveblocks.getRooms({ startingAfter });
+  pages.push(data);
+  startingAfter = nextPage;
+}
+```
+
+#### Liveblocks.createRoom [#post-rooms]
+
+Programmatically creates a new room from a room ID. The `defaultAccesses` option
+is required—setting it to `["room:write"]` creates a public room, whereas
+setting it to `[]` will create a private room that needs
+[ID token permission to enter](/docs/rooms/permissions/id-token). This is a
+wrapper around the
+[Create Room API](/docs/api-reference/rest-api-endpoints#post-rooms) and returns
+the same response.
+
+```ts
+const room = await liveblocks.createRoom("my-room-id", {
+  defaultAccesses: ["room:write"],
+});
+
+// { type: "room", id: "my-room-id", ... }
+console.log(room);
+```
+
+A number of room creation options are available, allowing you to set permissions
+and attach custom metadata.
+
+```ts
+const room = await liveblocks.createRoom("my-room-id", {
+  // The default room permissions. `[]` for private, `["room:write"]` for public.
+  defaultAccesses: [],
+
+  // Optional, the room's group ID permissions
+  groupAccesses: {
+    design: ["room:write"],
+    engineering: ["room:presence:write", "room:read"],
+  },
+
+  // Optional, the room's user ID permissions
+  userAccesses: {
+    "my-user-id": ["room:write"],
+  },
+
+  // Optional, custom metadata to attach to the room
+  metadata: {
+    myRoomType: "whiteboard",
+  },
+});
+```
+
+Group and user permissions are only used with
+[ID token authorization](/docs/api-reference/liveblocks-node#id-tokens), learn
+more about
+[managing permission with ID tokens](/docs/rooms/permissions/id-token).
+
+#### Liveblocks.getRoom [#get-rooms-roomId]
+
+Returns a room. Throws an error if the room isn’t found. This is a wrapper
```

Comment:
We throw our custom error called `LiveblocksError` if the room isn’t found. Should we mention this error class too somewhere?

```ts
if (!res.ok) {
  const text = await res.text();
  throw new LiveblocksError(res.status, text);
}
```

Reply:
Yes, good idea! Could you write a code snippet showing me the recommended way to display and type an error?